### PR TITLE
Improved: Ensure unique email when sendEmail OFBIZ-13170

### PR DESCRIPTION
**Pull request by nmalin**
When you run the service sendCommEventAsEmail if commEvent to send contains multiple party with the same email (through different contactMech), email will be sent to many times to the same box (depending on the smtp configuration).

To avoid this case, we will ensure that an email is unique for the sender list.

### CODE REVIEW

**Comment by JacquesLeRoux** [comment]
Did not test (not easy) but after simple review sounds good to me because of the use of `emailAlreadyLoad`

File: applications/party/src/main/java/org/apache/ofbiz/party/communication/CommunicationEventServices.java [hunk 1]
```diff
@@ -165,45 +165,35 @@ public static Map<String, Object> sendCommEventAsEmail(DispatchContext ctx, Map<
                     return ServiceUtil.returnError(errMsg + " " + communicationEventId);
                 }
 
-                // add other parties from roles
-                String sendCc = null;
-                String sendBcc = null;
+                // add other parties from roles, collect all email on map to parse it after
+                List<String> emailAlreadyLoad = UtilMisc.toList(sendTo);
```
Comment (gtchaboussie) [inline applications/party/src/main/java/org/apache/ofbiz/party/communication/CommunicationEventServices.java]: alreadyLoaded instead of alreadyLoad ?

**Comment by nmalin** [comment]
Thanks both for remarks
@JacquesLeRoux yes it's hard to test however we have some integration test that cover any regression :)

**Comment by JacquesLeRoux** [comment]
👍

**Pull request merged**